internal/runtime/client/client.go

```diff
@@ -50,15 +53,17 @@ const defaultDiscoveryTimeout = 10 * time.Second
 
 // Options are creation options for a Client.
 type Options struct {
-	Catalog  *runtimecatalog.Catalog
-	Registry runtimeregistry.ExtensionRegistry
+	Catalog    *runtimecatalog.Catalog
+	Registry   runtimeregistry.ExtensionRegistry
+	Kubeclient ctrlclient.Client
```

Comment:
Can't / shouldn't use client because we already have types of that name - it would be good to be consistent, but that would mean changing client / Client (for the import too!)

Reply:
That's strange, don't we have that issue in other cases? (I think for import it's just a warning)

Reply:
For the import I get a compile error on `client` - it's referenced inside the `client` struct, not sure if that's having an impact.

Reply:
```go

var _ Client = &client{}

type client struct {
	catalog  *runtimecatalog.Catalog
	registry runtimeregistry.ExtensionRegistry
	Client   client.Client
}
```
This is a complile error for me.

Reply:
Ah I see. I was suggesting to only change the field name not the import (in Options and client).
The import has to stay the same

Reply:
+1 to rename the field Client